src/common/showToast.js

```diff
@@ -0,0 +1,3 @@
+import Toast from 'react-native-simple-toast';
```

Comment:
Moved it to `utils.`
In future we can add more functions. For now one is sufficient.